nix/test-environments/example-setup-01.nix

```diff
@@ -113,11 +112,41 @@ in
               peg-tolerance-percentage = 0.2;
             }
           ];
+          contracts = [
+            {
+              name = "HistoricalDataFeedStoreV2";
+              address = upgradeableProxyContractAddressSepolia;
+              creation-byte-code = null;
+              deployed-byte-code = null;
+              contract-version = 1;
+            }
+            {
+              name = "multicall";
+              address = "0xcA11bde05977b3631167028862bE2a173976CA11";
+              creation-byte-code = null;
+              deployed-byte-code = null;
+              contract-version = 3;
```

Comment:
What is version 3?

I kinda have idea for 1 and 2. Is 3 related to contracts that are not deployed/developed by us or?

Reply:
3 comes from their documentation ( and obviously, some day we may have logic based on this number). Currently we use `agreggate` contract call which can be also found in previous version of `multicall`. 

[Multicall3](https://github.com/mds1/multicall3/blob/main/src/Multicall3.sol) is deployed on over 250 chains at 0xcA11bde05977b3631167028862bE2a173976CA11. The full list of deployed chains along with the Multicall3 ABI can be found at https://multicall3.com/. The ABI is provided in various formats, and can be copied to your clipboard or downloaded to a file.